AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -79,28 +79,30 @@ private extension SessionHeaderView {
     }
     
     var sensorType: some View {
-        var sensorNames = [String]()
+        var stream = [String]()
         var text = ""
-        for session in session.allStreams! {
+        
+        session.allStreams!.forEach { session in
             if var name = session.sensorPackageName {
-                if name.contains(":") {
-                    name = name.components(separatedBy: ":").first!
-                } else {
-                    name = name.components(separatedBy: "-").first!
-                }
+                componentsSeparation(name: &name)
```

Comment:
for the future: please use verbs to name functions. In this case it would be `separateComponents(name: &name)`